src/ert/run_models/base_run_model.py

```diff
@@ -406,7 +405,9 @@ def get_current_status(self) -> dict[str, int]:
                 for real in all_realizations.values():
                     status[str(real["status"])] += 1
 
-        status["Finished"] += self.active_realizations.count(False)
+        status["Finished"] += self.active_realizations.count(
```

Comment:
It would be nice to have a small comment / explanation here.

Reply:
Yes, either a comment or put it in a separate function and have a unit test on it.
The problem was that when we added all the `self.active_realizations.count(False)`, we also add the realizations that have deselected from the run. 
The idea was to use the the original `'active_realizations'`-list (which was already an instance variable `self_initial_realizations_mask`), and subtract this from the previous number.